### PR TITLE
CIP-1855 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #728: CIP-1855 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #729.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #728.

([updated proposal](https://github.com/rphair/CIPs/blob/remediate-1855/CIP-1855/README.md))

### CODE REVIEW

File: CIP-1855/README.md [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-1855
 Status: Active
-Type: Standards
+Category: Wallets
+Authors:
+  - Samuel Leathers <samuel.leathers@iohk.io>
+  - John Lotoski <john.lotoski@iohk.io>
+  - Michael Bishop <michael.bishop@iohk.io>
+  - David Arnold <david.arnold@iohk.io>
+Implementors: N/A
```
Comment (Ryun1) [inline CIP-1855/README.md:11]: Unsure about the extent of implementations

**Review by Ryun1 — APPROVED** [review]
LGTM

I would query the active status, since I am unsure about the extent of implementations, I am unable to find any.

**Comment by rphair** [comment]
@Ryun1 @KtorZ this is what I used as evidence of more-or-less universal adoption: https://github.com/cardano-foundation/CIPs/pull/256#issue-1232625012
> 📍 **Mark CIP-1852, CIP-1853 & CIP-1855 as active.**
CIP-1852 (and its extensions, CIP-1853 + CIP-1855) is describing (has
been for a while) the effective derivation schemes used by all
desktop, light and hardware wallets operating in the Cardano ecosystem.

File: CIP-1855/README.md (hunk 1 above)
Comment (rphair) [reply CIP-1855/README.md:11]: Somehow we recorded in our CIP editor communications that "everybody implements it" (continuing in front-line discussion with https://github.com/cardano-foundation/CIPs/pull/729#issuecomment-1873538406)

**Comment by Ryun1** [comment]
> @Ryun1 @KtorZ this is what I used as evidence of more-or-less universal adoption: https://github.com/cardano-foundation/CIPs/pull/256#issue-1232625012

I have responded over on the over PR [#725 comment](https://github.com/cardano-foundation/CIPs/pull/725#discussion_r1439579546).

**Comment by rphair** [comment]
Maybe then we can demote this to `Proposed` if it turns out that there are in fact no implementations, contrary to the `Active` status designated with that long-ago comment: but first let's wait to see if @Crypto2099 knows of any...

File: CIP-1855/README.md (hunk 1 above)
Comment (KtorZ) [reply CIP-1855/README.md:11]: It'd rather be [] here IMO, as this is something implementable but we are simply not aware of any implementor.

**Comment by rphair** [comment]
@KtorZ (re https://github.com/cardano-foundation/CIPs/pull/729#discussion_r1441558235) I agree; I originally put that because https://github.com/cardano-foundation/CIPs/pull/256#issue-1232625012 suggested the somewhat opposite (generally implemented without referencing specific implementations).

It's set to `[]` now along with _Acceptance Criteria_ to identify 3 implementations as suggested by @Ryun1 for the similarly pending CIP-1853 (https://github.com/cardano-foundation/CIPs/pull/725#discussion_r1439059083): https://github.com/cardano-foundation/CIPs/pull/729/commits/d77aec81bbffd18c951d9da39a26cc9fecbc6890

File: CIP-1855/README.md (hunk 1 above)
Comment (rphair) [reply CIP-1855/README.md]: OK, set back to `Proposed` and implementors = the empty set. https://github.com/cardano-foundation/CIPs/pull/729#issuecomment-1878045424

**Review by KtorZ — APPROVED** [review]

**Pull request merged**